### PR TITLE
i1-game teams collection

### LINKED ISSUE #16: Create Game Team Fixture File
Create a truncated version of the game team raw data set that we can use for our tests.

Should be located at test/fixtures/game_teams.csv

### PR DESCRIPTION
**Pull request by hashmaster3k**
Created game teams collection class, test file and fixture file. Tests pass when used with game_teams.rb

### CODE REVIEW

File: test/game_teams_collection_test.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    gtc = GameTeamsCollection.new('./test/fixtures/game_teams.csv')
+    assert_instance_of GameTeamsCollection, gtc
+  end
+
+  def test_has_a_path
+    gtc = GameTeamsCollection.new('./test/fixtures/game_teams.csv')
+    assert_equal './test/fixtures/game_teams.csv', gtc.path
+  end
+
+  def test_it_can_read
```
Comment (dhalverson) [inline test/game_teams_collection_test.rb:19]: We'll probably want to tighten up our testing here, and make sure the other files have similar testing and testing for edge cases, and proper reading/creation of the instances.

Looks good though!

**Review by dhalverson — APPROVED** [review]
This looks good, we'll want to review the other classes to ensure they are the same as well.

**Comment by dhalverson** [comment]
This looks good, lets review the other classes to ensure consistency across. We should be able to move to methods soon!
